python/paddle/fluid/layers/tensor.py

```diff
@@ -884,3 +884,39 @@ def zeros_like(x, out=None):
         type='fill_zeros_like', inputs={'X': [x]}, outputs={'Out': [out]})
     out.stop_gradient = True
     return out
+
+
+def diag(diagonal):
```

Comment:
Change `diagonal` to `x` or `input`?

Reply:
x or input is good, but `diagonal` is more specific about the meaning of this input, so I think we may just leave it as it is.